### PR TITLE
manifold-viewer: Add version 9.0.174.2

### PR DESCRIPTION
**Pull request by rkolka**
Manifold Viewer is the free, read-only version of Manifold Release 9 - a GIS, ETL, data science and DBMS tool. 
Manifold official builds are issued about every 4-6 months. Cutting Edge builds are issued monthly or semimonthly.

### CODE REVIEW

File: bucket/manifold-viewer-9-official.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "version": "9.0.174",
+    "description": "Official build of Manifold Viewer Release 9",
+    "homepage": "http://manifold.net/",
+    "license": "Unknown",
+    "depends": "extras/vcredist2017" ,
```
Comment (Ash258) [inline bucket/manifold-viewer-9-official.json:6]: Never depends on vcredists

File: bucket/manifold-viewer-9-official.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "url": "http://www.manifoldgis.com/updates/manifold-viewer-9.0.174-x64.zip",
+    "extract_dir": "manifold-viewer-9.0.174-x64",
+    "hash": "sha256:23e85f8b1320c161ef6288f65c1ae56f5098dabbe2d9d98814c7f6dd085843b4",
```
Comment (Ash258) [inline bucket/manifold-viewer-9-official.json:9]: Exchange position of these two properties

File: bucket/manifold-viewer-9-official.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+    "version": "9.0.174",
+    "description": "Official build of Manifold Viewer Release 9",
+    "homepage": "http://manifold.net/",
+    "license": "Unknown",
+    "depends": "extras/vcredist2017" ,
+    "url": "http://www.manifoldgis.com/updates/manifold-viewer-9.0.174-x64.zip",
```
Comment (Ash258) [inline bucket/manifold-viewer-9-official.json:7]: This is clearly intended as 64bit only

File: bucket/manifold-viewer-9-official.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{
+    "version": "9.0.174",
+    "description": "Official build of Manifold Viewer Release 9",
```
Comment (Ash258) [inline bucket/manifold-viewer-9-official.json:3]: This is nonnse description of application. Does not explain what it is for

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Review by Ash258 — COMMENTED** [review]
Manifest name is commpleletely nonsense. Why tehre is version in the name? Remove the official

File: bucket/manifold-viewer-9-official.json (hunk 1 above)
Comment (rkolka) [reply bucket/manifold-viewer-9-official.json:6]: ok. (but why? the ability to declare dependencies was appealing. I've noticed that vcredist2017 scoop install acts wierd. Is that the reason?)

File: bucket/manifold-viewer-9-official.json (hunk 2 above)
Comment (rkolka) [reply bucket/manifold-viewer-9-official.json:9]: ok.

File: bucket/manifold-viewer-9-official.json (hunk 3 above)
Comment (rkolka) [reply bucket/manifold-viewer-9-official.json:7]: well, this really includes both versions. Running 32bit version might be needed even on 64bit machine for compatibility with other software http://www.manifold.net/doc/mfd9/launch_in_32-bit_mode.htm

File: bucket/manifold-viewer-9-official.json (hunk 4 above)
Comment (rkolka) [reply bucket/manifold-viewer-9-official.json:3]: it is tautology, true. Will fix

**Comment by rkolka** [comment]
> Why tehre is version in the name? 
Manifold refers to its products with such names. It may look unusual, true. R8 is also "current".
http://www.manifold.net/info/manifold8.shtml
http://www.manifold.net/info/manifold9.shtml

> Remove the official
Ok. I planned to submit manifests for "Cutting Edge" builds (again Manifold term). I wanted to distinguish. I guess "Cutting Edge" builds should live in versions repo or smth?

File: bucket/manifold-viewer-9-official.json (hunk 3 above)
Comment (rkolka) [reply bucket/manifold-viewer-9-official.json]: Oh, should I put this url inside   "architecture": {  "64bit": { }}  ?

**Comment by issaclin32** [comment]
I updated the manifest by myself because the PR has not been updated in a while.

**Review by rashil2000 — APPROVED** [review]

**Pull request merged**